src/main/java/service/strategy/hendlers/ReturnOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package service.strategy.hendlers;
+
+import data.base.Storage;
+import model.FruitTransaction;
+import service.strategy.OperationHandler;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void updateNumberOfFruit(FruitTransaction fruitTransaction) {
+        Integer sumOfFruitAfterReturn = Integer.sum(
+                Storage.STORAGE.get(fruitTransaction.getFruit()),
```

Comment:
It's better to use local variables for Storage.STORAGE.get(fruitTransaction.getFruit() and  fruitTransaction.getQuantity() to increase readability